core/client/app/templates/setup/three.hbs

```diff
@@ -12,7 +12,8 @@
 sally.sanders@example.com" value=users}}
     </form>
 
-    <button {{action "signin"}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
+    <button {{action 'invite'}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
         {{buttonText}}
     </button>
+    <a {{action 'signin'}} href="#" class="gh-flow-skip">I'll do this later, take me to my blog!</a> 
```

Comment:
If it can be done per the spec, that would be preferable. If that is going to be really annoying/difficult (@jaswilli confirm?)  -- this solution is an acceptable alternative :+1: